Dashboard/Server/Summary/ChatProcessor.cs

```diff
@@ -0,0 +1,291 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Linq;
+using System.Text.RegularExpressions;
+
+namespace Dashboard.Server.Summary
+{
+	/// <summary>
+	/// Exception for empty string input for summarizer
+	/// </summary>
+	public class EmptyStringException : Exception
```

Comment:
Is this exception details known to Rajeev and UX team?

Reply:
No it is only for within the summarizer.